lib/trento/domain/cluster/cluster.ex

```diff
@@ -438,4 +469,42 @@ defmodule Trento.Domain.Cluster do
       %ClusterHealthChanged{cluster_id: cluster_id, health: new_health}
     end
   end
+
+  defp handle_hosts_execution_data(
```

Comment:
nitpick: i would vouch against the usage of the word "data" in the domain logic. Had to scroll to the function to see what it was actually about, maybe we can use a name which is more "domain related", such as "emit_host_execution_completed_event".